en/docs/deploy/deployment-best-practices/runtime-security.md

```diff
@@ -0,0 +1,99 @@
+# Runtime Security
+
+### 1. Apply Security Patches Regularly
+
+* **VSCode Plugin**: Always use the latest release for `WSO2 Integrator: BI` and `Ballerina` VSCode plugin.  
```

Comment:
https://github.com/wso2/docs-bi/pull/169/files#r2519043396 Same appears in all the places.